### DISCUSSION TITLE
Inline plugin - unable to scroll the page with mouse wheel in desktop Safari

### DISCUSSION THREAD

**Discussion by Andr3wTayl0r** [igordanchenko/yet-another-react-lightbox#255] [Q&A] [answered]
When I hover over an inline Lightbox in desktop Safari (17.3.1) I am unable to scroll the page down - but move the pointer to somewhere other than the Lightbox and scrolling works.  This looks like a Safari issue as Chrome does not have this problem.

Try doing that here: [https://yet-another-react-lightbox.com/plugins/inline](https://yet-another-react-lightbox.com/plugins/inline)

Any ideas as to a fix or work-around?

Thanks

**Comment by igordanchenko** [comment]
Are you using a mouse by any chance? I'm unable to reproduce the issue with a trackpad (Safari 17.4.1)

**Reply by Andr3wTayl0r** [reply]
ah - yes I'm using a mouse - works fine with a trackpad - thanks

**Reply by igordanchenko** [reply]
OK. Let me find a mouse and I'll see what's going on here.

**Reply by igordanchenko** [reply, marked as answer]
Wow, this is quite a peculiar quirk in Safari. The bug reproduces only when __all__ of the following conditions are met:
1) the element has `overflow: hidden`
2) the element has `overscroll-behavior: contain`
3) the element has a non-passive wheel event listener
```js
element.addEventListener("wheel", () => {}, { passive: false })
```

I highly doubt it's reasonable to file this bug with WebKit, as it may take a decade for someone to investigate it.

Anyway, I'll update the Inline plugin and set `overscroll-behavior-y` to `auto`. It should address the issue. I'll release the fix later today.

**Reply by Andr3wTayl0r** [reply]
Thanks soooo much!
